proposals/0540-jsem.rst

```diff
@@ -0,0 +1,342 @@
+
+jsem: parallelism semaphores for GHC
```

Comment:
My idea was more that the work “semaphore” is strange in this context, it's an implementation mechanism rather than a description of the feature. I would probably look at something along the idea of “job server”. Or maybe “shared jobs”.

But, as you rightly point out, it's not worth losing hair (and more importantly time) over.

Reply:
The flag is likely only used by tools? Then a long descriptive flag doesn't hurt. `-juse-jobserver` maybe?